### ISSUE TITLE
Syntax highlighting troubles inside strings

### ISSUE THREAD

**Issue by Kolfrad** [Eforen/sublime-syntax-sprak#1]
Hey there !
Before anything I'd like to thank you for your awesome plugin, really helpful for those who want to code a full OS (Or anything big) without having to use the nice but limited game editor.

Not sure if you already spotted it but the syntax highlighting goes wrong when you write statements, integers and some symbols inside a string. Some examples : 

```
var int = 12   <- 12 will be colored as an integer
if a == b      <- if and == will be as statements
#comment       <- This will be colored as a comment

Print("Found 3 files")     <- 3 will be colored as an integer
Print("if you want")       <- if will be colored as a statement
Print("###Main Menu###")   <- All the line after the first # will be colored as a comment
                              and will break further highlighting
```

After the last example, everything turns the same color (Yellow for monokai users) as if it was a string (Probably because the plugin overlook the closing quotes of the string) except function names, statements and integers.

Aside of this minor problem, I'm glad I can finally have some space and functionality to break my game.
I don't have the skills to come up with a solution but I hope someone will manage to fix this little hiccup.

**Comment by Eforen** [comment]
I am glad you like it. Honestly I did see it and I was unsure if anyone else would even like the plugin and am currently sick. But knowing atleast one person is enjoying it I will try to get that bug smashed out tomorrow :)

**Comment by Eforen** [comment]
This issue should be resolved now I am tagging 1.0.1 so package.io should pull the new version and you should be able to update your version.

**Comment by Eforen** [comment]
It is now live on PackageControl.io
